gno.land/cmd/gnoland/main.go

```diff
@@ -105,6 +106,13 @@ func (c *gnolandCfg) RegisterFlags(fs *flag.FlagSet) {
 		"localhost:26657",
 		"replacement for '%%REMOTE%%' in genesis",
 	)
+
+	fs.Int64Var(
+		&c.maxCycles,
+		"max-vm-cycles",
+		10*1000*1000,
```

Comment:
just a small suggestion :)

```suggestion
		10_000_000,
```

Reply:
TIL